content/en/watchdog/insights.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Watchdog Insights
+kind: documentation
+description: View anomalies and outliers in context with Watchdog Insights
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-watchdog-insights-log-management/"
+  tag: "Blog"
+  text: "Augmented troubleshooting with Watchdog Insights"
+- link: "https://www.datadoghq.com/blog/watchdog-insights-apm/"
+  tag: "Blog"
+  text: "Automatically detect error and latency patterns with Watchdog Insights for APM"
+---
+
+## Overview
+
+Datadog Watchdog constantly runs in the background, scanning for anomalies in your organization's entire data set. As you navigate the Datadog UI, Watchdog Insights show you the information most relevant to your current context. Using your active product and your active query, Watchdog Insights displays a filtered and sorted list of anomalies.
+
+Investigating an incident requires trial and error. Drawing from their experience, engineers familiar with a particular area know where to first look for potential problems. Using Watchdog Insights allows all engineers, including less experienced ones, to pay attention to the most important data and accelerate their incident investigations.
+
+Each insight highlights one outlier or anomaly affecting a subset of users. Depending on the product area, Watchdog Insights displays different types of anomalies. Examples include, but are not limited to, the following:
+- Error outliers
+- Latency outliers
+- Log anomalies
+- Lock pressure
+- Deadlocked threads
```

Comment:
Samir, please change this list to your suggested examples.

Reply:
Let's use the most descriptive title for Insights:
"
- Error and Latency Outliers in Logs, Traces and RUM Views
- Spike in error logs
- New error logs
- Deadlocked threads
- High % of unready kubernetes pods
"